### PR TITLE
Add a default config file with a set of rules

### PR DESCRIPTION
**Pull request by perruche**
Based on the [full config file provided by tailwind](https://github.com/tailwindcss/tailwindcss/blob/master/stubs/defaultConfig.stub.js)

**What is different from the original default config file ?** 
- Smaller set of rules activated (up to debate what should be part of it)
- Rules order by thematic rather than alphabetical order
- Breakpoints (or screens as called in tailwind) use the same values & syntax as the scss-toolkit
- Smaller set of variants activated

Feedback requested from @studiometa/css

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,227 @@
...
+     * Size
+     */
+    container: {},
+    width: theme => ({
+      auto: 'auto',
+      ...theme('spacing'),
+      full: '100%',
+      screen: '100vw',
+    }),
+    maxWidth: (theme, { breakpoints }) => {
```
Comment (titouanmathis) [inline src/index.js]: I am not sure the `max-width` utilities will be used. Maybe we could remove them in this first draft?

File: src/index.js [hunk 2]
```diff
@@ -1,5 +1,227 @@
...
+     */
+    borderRadius: {},
+    borderWidth: {},
+
+    /**
+     * Layer
+     */
+    zIndex: {
+      goku: '9000',
+      default: '1',
```
Comment (titouanmathis) [inline src/index.js]: What about adding the following indexes : 
- `above` with a value of `2`
- `under` with a value of `-1`

Any thoughts @studiometa/css  ?

File: src/index.js [hunk 3]
```diff
@@ -1,5 +1,227 @@
...
+    gridRowEnd: {},
+
+    /**
+     * Transforms
+     */
+    rotate: {},
+    scale: {},
+    skew: {},
+    transformOrigin: {},
+    translate: (theme, { negative }) => ({
```
Comment (titouanmathis) [inline src/index.js]: Nice! 👍

File: src/index.js [hunk 4]
```diff
@@ -1,5 +1,227 @@
...
+ */
 module.exports = {
-  theme: {},
-  variants: {},
+  theme: {
+    /**
+     * Breakpoints
+     */
+    screens: {
+      xxs: '0px',
```
[the remark this answers is not in the document]
Comment (perruche) [reply src/index.js]: yeah that's probably a good idea, haven't thought of that

File: src/index.js (hunk 2 above)
Comment (perruche) [reply src/index.js]: 👍

Comment (ThomasCtnt) [reply src/index.js]: Great idea, i used to add them in the config of the toolkit we used and it's pretty useful 👍

**Pull request merged**